index.html

```diff
@@ -53,6 +53,230 @@ <h1>I'm Anold. <br>Glad to see <br> you!</h1>
+            </li>
+            <li>
+              <img src="images/Counter.png" alt="counter image">
+              <span>2015</span>
+            </li>
+          </ul>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="languages">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <button>See Project</button>
+      </article>
+    </section>
+
+    <section class="about_me">
+      <article class="about_myself">
+        <h1>About Myself</h1>
+        <p class="tomato">Hello I'm a software developer! I can help you build a product , feature or website. Look through some of my
+          work and
+          experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        <article class="social">
+          <div class="margin_sm lets_connect">
+            <a href="#">LET'S CONNECT</a>
+          </div>
+          <ul class="social_icons">
+            <li>
+              <a href="#"><img src="images/twitter.svg" alt="twitter icon"></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/Linkedin icon-1.svg" alt="Linkedin icon"></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/medium.svg" alt="medium icon"></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/github.svg" alt="github icon"></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/angellist.svg" alt="angellist icon"></a>
+            </li>
+          </ul>
+          <button class="my_resume margin_sm">Get my resume</button>
+
+          <!--technologies-->
+          <div class="technologies">
+            <ul class="technologies_container">
+              <li>
+                <div class="tech_container">
+                  <h6>Languages</h6>
+                  <div class="tech_img">
+                    <img src="images/Disabled.png" alt="arrow down icon">
+                  </div>
+                </div>
+                <ul class="technologies_container-languages">
+                  <!--languages-->
+                  <li>
+                    <div>
+                      <img src="images/js icon.png" alt="javascript icon">
+                    </div>
+                    <div>
+                      <p>JavaScript</p>
+                    </div>
+                  </li>
+                  <li>
+                    <div>
+                      <img src="images/html icon.png" alt="html icon">
+                    </div>
+                    <div>
+                      <p>HTML</p>
+                    </div>
+                  </li>
+                  <li>
+                    <div>
+                      <img src="images/css icon.png" alt="css icon">
+                    </div>
+                    <div>
+                      <p>CSS</p>
+                    </div>
```

Comment:
Kindly use the right font styles for this section of your work, it should be similar to the design given in the figma template 😄

Your design :

![image](https://github.com/MPDADDY/my_portfolio/assets/20537712/53d612a0-bbdc-40a2-8897-afe6e405fc5a)

Figma design :
![image](https://github.com/MPDADDY/my_portfolio/assets/20537712/2d543d33-38ff-4736-bac0-77588f033db9)